src/core/structured-log.js

```diff
@@ -277,12 +277,15 @@
       invoke('TRACE', mt, arguments);
     };
 
-    self.information = function(messageTemplate) {
+    self.verbose = self.trace;
```

Comment:
I think the idea behind this change is good, but from a usability perspective, there are some caveats.

I often use e.g. the Chrome dev tools to explore what methods are on an object. Having `verbose()` and `debug()` will make it a bit confusing which is "correct" vs which is a patch to avoid typos.

This is also out of whack with the level names - it suggests there are levels named `DEBUG` and `VERBOSE` that can be used with `minimumLevel()`, but these don't exist, so saving bugs in one place might encourage bugs in another.

Just my 0.2c, but perhaps we could discuss this via a separate ticket?